firebase-vertexai/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+* [changed] **Breaking Change**: Migrated `FirebaseVertexAIException` from a sealed class to an abstract class, and marked constructor as internal. (#6368)
```

Comment:
Since it's _all_ the exception classes' constructors
```suggestion
* [changed] **Breaking Change**: Migrated `FirebaseVertexAIException` from a sealed class to an abstract class, and marked constructors as internal. (#6368)
```